### DISCUSSION TITLE
ESP32-C3 : How to enable AS608 ?

### DISCUSSION THREAD

**Discussion by Noschvie** [arendst/Tasmota#17695] [Q&A] [answered]
#9568 : Add fingerprint sensor 

`
#define USE_AS608              // Add support for AS608 optical and R503 capacitive fingerprint
`
`#define USE_AS608_MESSAGES     // Add verbose error messages
`

These lines added to user_config_override.h seems not to be working for the ESP32-C3. The settings "AS608 RX/TX" aren't available, but can be selected running for Wemos D1 mini.
What is missing ? Thanks!

**Comment by barbudor** [comment, marked as answer]
What exact variant of tasmota did you tried to compile ?
For C3, you must compile `tasmota32c3` or ` tasmota32c3cdc`

Working fine on my side using latest development version

![image](https://user-images.githubusercontent.com/2996042/212436923-c261a84f-771a-4024-a34a-dce7def1634c.png)

![image](https://user-images.githubusercontent.com/2996042/212437001-ce416ea7-5d7e-40e0-8aaf-df7d8ec3062e.png)
![image](https://user-images.githubusercontent.com/2996042/212437082-5036a764-419e-4213-b48f-fddce0b233b2.png)

**Reply by Noschvie** [reply]
Used Docker-Tasmota to compile using "./compile.sh tasmota32c3"

Now a "git pull" solved my issue.
Thanks!
